pkg/maps/ctmap/ctmap_privileged_test.go

```diff
@@ -33,6 +40,23 @@ func (k *CTMapPrivilegedTestSuite) SetUpSuite(c *C) {
 	c.Assert(err, IsNil)
 }
 
+func Benchmark_MapBatchLookup(b *testing.B) {
+	m := newMap(MapNameTCP4Global+"_test", mapTypeIPv4TCPGlobal)
+	err := m.OpenOrCreate()
+	defer m.Map.Unpin()
```

Comment:
I'm planning on making pinning opt-in, could you call Unpin() here directly after OpenOrCreate? This acts as a reminder for me to remove all extraneous Unpin() calls from tests. Putting them behind a defer will make them fail silently. Thanks!

```suggestion
	assert.NoError(b, m.Map.Unpin())
```